src/workers/compiler.ts

```diff
@@ -7,7 +7,8 @@ import babelPresetSolid from 'babel-preset-solid';
 import { rollup, Plugin } from '@rollup/browser';
 import dd from 'dedent';
 
-export const CDN_URL = (importee: string) => `https://esm.sh/${importee}?dev`;
+// export const CDN_URL = (importee: string) => `https://esm.sh/v112/${importee}?dev`;
```

Comment:
Any chance you can remove the commented code? After that I'd be happy to merge this

Reply:
Just done it